gst/nnstreamer/nnstreamer_plugin_api_util_impl.c

```diff
@@ -1430,3 +1430,50 @@ nnstreamer_version_fetch (guint * major, guint * minor, guint * micro)
   if (micro)
     *micro = (NNSTREAMER_VERSION_MICRO);
 }
+
+/**
+ * @brief Compare dimension strings
+ * @return TRUE if equal, FALSE if given dimension strings are invalid or not equal.
+ */
+gboolean
+gst_tensors_dimension_string_equal (const gchar * dimstr1,
+    const gchar * dimstr2)
+{
+  tensor_dim dim1, dim2;
+  int rank1, rank2, i, j, num_tensors1, num_tensors2;
+  gchar **strv1;
+  gchar **strv2;
+  gboolean res = TRUE;
+
+  strv1 = g_strsplit (dimstr1, ",", -1);
+  strv2 = g_strsplit (dimstr2, ",", -1);
```

Comment:
I think It is `.`, not `,` to separates dims from tensors.

Reply:
Let's update it with JY's suggestion.

Reply:
Updated to use both `,` and `.` to separates tensors.
Thank you :)